index.js

```diff
@@ -118,12 +123,12 @@ function Gas (runner, options) {
 
   runner.on('hook end', () => { startBlock = sync.blockNumber() + 1 })
 
-  runner.on('pass', test => {
+  runner.on('pass', async test => {
     let fmt
```

Comment:
Unfortunately mocha doesn't actually let us run these callbacks as promises. But I *think* `deployAnalytics` should be safely sync . . . was there something going on that suggested this needs to be async and the analytics should be awaited?

Reply:
The reason I added this was because I needed `deployMap` to have the contracts populated before processing `methodAnalytics`.  I was running into a problem where the `deployMap` was empty so that i why I added this.  However, I just tested locally and it looks like it was another issue as it works without async

Reply:
@cgewecke just wanted to make sure you saw this as you were cleaning up

Reply:
Ok awesome. . .